scripts/ci-e2e-kind.sh

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+create_kind_cluster () {
+  # create a cluster with the local registry enabled in containerd
+  # add encryption config and the kms static pod manifest with custom image
+  cat <<EOF | kind create cluster --retain --image kindest/node:"${KUBERNETES_VERSION}" --name "${KIND_CLUSTER_NAME}" --wait 2m --config=-
+  kind: Cluster
+  apiVersion: kind.x-k8s.io/v1alpha4
+  containerdConfigPatches:
+  - |-
+    [plugins."io.containerd.grpc.v1.cri".registry.mirrors."localhost:${reg_port}"]
```

Comment:
Is this a requirement?

Reply:
yeah, this is required to enable local registry: https://kind.sigs.k8s.io/docs/user/local-registry/

Reply:
nvm